pkg/bundled/cloudconfigs/09_systemd_services.yaml

```diff
@@ -21,12 +21,7 @@ stages:
           - multi-user.target
           - getty@tty1
           - iscsid
-          - systemd-timesyncd
-          - nohang
-          - nohang-desktop
-          - fail2ban
```

Comment:
fail2ban was nice. can we just detect and install it if it's in the base image/available? As we default with ssh enabled, makes sense so harden it at least as much as possible.